src/components/Pages/Classes/List.vue

```diff
@@ -129,6 +136,17 @@ export default {
             this.editMode = true;
         },
 
+        selectClass(code, name) {
+            this.selectedClassCode = code;
+
+            this.$store.commit('setClassName', name);
+
+            window.localStorage.setItem('code', code);
+            window.localStorage.setItem('nameClass', name);
```

Comment:
Out of curiosity: was 'className' reserved in some way? If not, why the reverse order?

Reply:
@BarryOgg I change it to 'selectedClassName' because we use `className` in few places.